src/main/java/org/jenkinsci/plugins/workflow/steps/scm/SCMStep.java

```diff
@@ -110,19 +110,23 @@ public final void checkout(Run<?,?> run, FilePath workspace, TaskListener listen
                     changelogFile = Files.createTempFile(run.getRootDir().toPath(), "changelog", ".xml").toFile();
                 }
             }
+            long changelogOriginalModifiedDate = (changelogFile != null) ? changelogFile.lastModified() : 1L;
             SCM scm = createSCM();
             SCMRevisionState baseline = null;
             Run<?,?> prev = run.getPreviousBuild();
             if (prev != null) {
                 synchronized (prev) {
-                MultiSCMRevisionState state = prev.getAction(MultiSCMRevisionState.class);
-                if (state != null) {
-                    baseline = state.get(scm);
-                }
+                    MultiSCMRevisionState state = prev.getAction(MultiSCMRevisionState.class);
+                    if (state != null) {
+                        baseline = state.get(scm);
+                    }
```

Comment:
nit: Try not to modify lines that you don't need to, especially if you are just changing whitespace, both to make review easier (not as important for small PRs like this) and so that tools like `git blame` only see "real" changes, which is very helpful when trying to understand the history of a piece of code while investigating an issue.

Reply:
Would it be preferable to undo this formatting change, or has that ship already sailed?

Reply:
I think we'll probably end up squash-merging the PR, so you could still clean it up, but it's not a big deal or anything.